static/html/ldap-login.html

```diff
@@ -13,7 +13,7 @@ <h2 class="heading">Log in to Your Account</h2>
     <div class="form-row">
       <div class="input-desc">
         <label for="password">Password</label>
-        <span class="subtle-text input-label-right">Forgot? <a href="/send-reset-password?session_key={{ .SessionKey }}">Reset Password</a></span>
+        <span class="subtle-text input-label-right">Forgot? <a href="{{ absPath }}/send-reset-password?session_key={{ .SessionKey }}">Reset Password</a></span>
```

Comment:
Isn't this wrong? E.g. if absPath returns `/hi/` the URL path will be `/hi//send-reset...`